app/builders/content_change_email_builder.rb

```diff
@@ -53,7 +53,6 @@ def body(content_change, subscriptions, address)
 
   def presented_content_change(content_change, subscriptions)
     copy = ContentChangePresenter.call(content_change)
-    return copy if subscriptions.empty?
 
     subscriber_list = subscriptions.first.subscriber_list
```

Comment:
Are we confident there isn't any risk that this is lazily loaded and could end up with an empty array? I can't recall the load ordering but I wondered if there's a rare risk that someone has unsubscribed at the point of generating the email and we end loading up no active, immediate subscriptions. Hopefully not.